num2words/lang_PL.py

```diff
@@ -239,12 +245,14 @@ def to_currency(n, currency='EUR', cents=True, seperator=','):
             minus = False
 
         n = abs(n)
-        left = n / 100
+        left = n // 100
         right = n % 100
     else:
         n = str(n).replace(',', '.')
         if '.' in n:
             left, right = n.split('.')
+            if len(right)==1:
+                right = right+'0'
```

Comment:
Could you add whitespace around the operators? So we keep respecting PEP8?
`if len(right) == 1:` and `right = right + '0'`